matchms/importing/load_from_msp.py

```diff
@@ -149,16 +147,20 @@ def parse_metadata(rline: str, params: dict):
     matches = []
```

Comment:
Params is not returned, but edited in the function and used outside the function again. This might slightly improve speed, but I think this does not outweight the downside.

Suggestion:
Just return params and use the output in other functions.